Update README.md to clarify engineering engagement during drafting (#27449)
fleetdm/fleet: handbook/product-design/README.md

6. Draft changes to the Fleet product that solve the problem specified in the story.
   - Constantly place yourself in the shoes of a user while drafting changes.
   - Use dev notes (component available in our library) to highlight important information to engineers and other teammates. - Reach out to sales, customer success, and demand for a business perspective.

>    - At the first design review of each design sprint stories will be reviewed and prioritized by product and engineering. An engineering DRI will be assigned to each ticket and will help determine whether additional engineering research stories need to be written and pointed.

Author: ```suggestion
   - During the Design sprint kickoff meeting, Engineering managers will determine whether additional engineering research stories need to be written and pointed, which will be filed during the meeting so they can be estimated and pulled into next engineering sprint. 
```

>        - At the first design review of each design sprint stories will be reviewed and prioritized by product and engineering. An engineering DRI will be assigned to each ticket and will help determine whether additional engineering research stories need to be written and pointed.

Author: ```suggestion
   - At the first design review of each design sprint stories will be reviewed and prioritized by product and engineering. An engineering DRI will be assigned to each ticket and will help determine whether additional engineering research stories need to be written and pointed.
```

   - Constantly place yourself in the shoes of a user while drafting changes.
   - Use dev notes (component available in our library) to highlight important information to engineers and other teammates. - Reach out to sales, customer success, and demand for a business perspective.

>    - Engage engineering to gain insight into technical costs and feasibility.

Author: ```suggestion

```
